packages/web-twig/src/Resources/components/Toast/ToastBar.twig

```diff
@@ -0,0 +1,60 @@
+{# API #}
+{%- set props = props | default([]) -%}
+{%- set _closeLabel = props.closeLabel | default('Close') -%}
+{%- set _color = props.color | default('inverted') -%}
+{%- set _hasIcon = props.hasIcon | default(false) -%}
```

Comment:
don't we want iconName as well?

Reply:
I dismissed it originally because it's not in Figma. But the Figma version of `Alert` doesn't have it either, so, at the end of the day, I think we can match the `Alert` Twig/React implementation with this.